generator/src/main/java/org/corfudb/generator/distributions/Operations.java

```diff
@@ -0,0 +1,42 @@
+package org.corfudb.generator.distributions;
+
+import com.google.common.collect.ImmutableList;
+import org.corfudb.generator.State;
+import org.corfudb.generator.operations.Operation;
+import org.corfudb.generator.operations.OptimisticTxOperation;
+import org.corfudb.generator.operations.ReadOperation;
+import org.corfudb.generator.operations.RemoveOperation;
+import org.corfudb.generator.operations.SleepOperation;
+import org.corfudb.generator.operations.SnapshotTxOperation;
+import org.corfudb.generator.operations.WriteOperation;
+
+import java.util.List;
+
+/**
+ * This class implements a distribution of all possible operations that the generator
+ * can execute.
+ *
+ * Created by maithem on 7/14/17.
+ */
+public class Operations extends DataSet {
+
+    final private List<Operation> operations;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename field "operations" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1700)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)